web/frontend/src/components/slam-2d-render.vue

```diff
@@ -110,16 +192,47 @@ const updateCloud = (pointcloud: Uint8Array) => {
   intersectionPlane.position.set(center.x, 0, center.z);
   raycaster.objects = [intersectionPlane];
 
+  // construct grids
+  const axesHelper1 = new THREE.AxesHelper( 5 );
+  axesHelper1.position.set(center.x, 0, center.z);
+  axesHelper1.rotateY(Math.PI/2)
+  axesHelper1.scale.x = 1e5
+  axesHelper1.scale.z = 1e5
+  axesHelper1.renderOrder = 998
+  
+  const axesHelper2 = new THREE.AxesHelper( 5 );
+  axesHelper2.position.set(center.x, 0, center.z);
+  axesHelper2.rotateY(-Math.PI/2)
+  axesHelper2.scale.x = 1e5
+  axesHelper2.scale.z = 1e5
+  axesHelper2.renderOrder = 997
+
+  const gridHelper = new THREE.GridHelper( 1000, 100, 0xCACACA, 0xCACACA ); // this needs to be updated so it is set to 1m
+  gridHelper.position.set(center.x, 0, center.z);
+  gridHelper.renderOrder = 996;
+
+  // add objects to scene  
+  scene.add(axesHelper1);
+  scene.add(axesHelper2);
+  scene.add(gridHelper);
   scene.add(points);
-  scene.add(marker);
   scene.add(intersectionPlane);
+  updatePose(props.pose!)
```

Comment:
did this because a refresh or toggle on card removed the base marker from the screen